README.md

```diff
@@ -41,231 +41,383 @@ This project contains two separate providers that operate in tandem:
 - kubectl version v1.11.3+.
 - Access to a Kubernetes v1.11.3+ cluster.
 
-### Installation
-To configure clusterctl with the OpenShift Agent providers, edit `~/.cluster-api/clusterctl.yaml` and add the following:
+# Usage
+
+## Prerequisites
```

Comment:
very minor but prereqs is also defined up here too https://github.com/openshift-assisted/cluster-api-provider-openshift-assisted/pull/172/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R38

Was it intended to be completely different?